homeassistant/components/local_todo/strings.json

```diff
@@ -8,6 +8,9 @@
           "todo_list_name": "To-do list name"
         }
       }
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
```

Comment:
Maybe already configured service fits better than account?

Reply:
Thanks, didn't even notice that.